tests/chart_tests/test_astronomer_registry.py

```diff
@@ -155,3 +155,18 @@ def test_astronomer_registry_statefulset_with_scc_disabled(self, kube_version):
             ],
         )
         assert len(docs) == 0
+
+    def test_astronomer_registry_statefulset_with_podSecurityContext_disabled(
```

Comment:
Now that `.Values.registry.podSecurityContext` is configurable we also need a test for the default case.